// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package demo;

service GetNameService {
<<<<<<< HEAD
  rpc GetID (IDRequest) returns (IDReply) {}
}

message IDRequest {
  int32 id = 1;
}

message IDReply {
=======
  rpc GetName (NameRequest) returns (NameReply) {}
}

message NameRequest {
  int32 id = 1;
}

message NameReply {
>>>>>>> de988bdddb4acfb3a9a210da2348bcff788ee20e
  string name = 1;
}
